### PR TITLE
platform support policy

### PR DESCRIPTION
**Pull request by MichaelBaj**
docs-21

### CODE REVIEW

File: docs/about_supported_platforms.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+- ***Platform*** - A physical or virtual system, typically consisting of some amount of resources for compute, memory, disk, and networking. *Example*: A physical server, virtual machine, or public cloud virtual machine instance.
+- ***Certification*** - The process of testing to ensure that a specific combination of a 128T software version and platform build meet requirements for functionality, stability, and performance. *Example*: The [128T Certification Program](#128t-certification-program) provides certification of a set of platforms with 128T software.
+- ***Certified Platform*** - A platform that is currently in the 128T Certification Program as a [128T Certified Plus Platform](#128t-certified-plus-platform), or [128T Certified Platform](#128t-certified-platform). *Example*: A specified build of Lenovo SR530 server is a certified platform.
+- ***Compatible Platform*** - A platform that meets or exceeds [minimum platform specifications](#minimum-platform-specifications) and is known to be able to run 128T software. *Example*: A Compulabs fitlet2 with Intel Atom E3950, 8GB of memory, and 128GB of disk is a compatible platform.
```
Comment (ptimmons) [inline docs/about_supported_platforms.md:14]: CompuLab

File: docs/about_supported_platforms.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+## 128T Certification Program
+
+128 Technology continuously works to maintain a platform abstraction in its software design. At the same time, there are often characteristics and optimizations that require platform-specific support. This support comes by way of a platform certification program which provides assurances of performance and function of the 128T software. For example, the 128T software utilizes [DPDK](https://www.dpdk.org) for broad driver support, and in some cases specific firmware revisions need to be loaded prior to being able to fully utilize 128T's optimized packet forwarding capabilities.
+
+Inclusion in the 128T Certification Program is based on a given platforms ability to continuously meet certain test and support criteria. New platforms are added to the program at 128 Technology's discretion. For details on how to get new platforms in to the 128T certification program, or further information regarding the program, please contact 128 Technology.
+
+### 128T Certified Plus Platform
+
+The 128T Certification program provides a designation of Certified Plus for certain platforms. Certified Plus platforms have the following:
```
Comment (ptimmons) [inline docs/about_supported_platforms.md:25]: ... have the following characteristics:

File: docs/about_supported_platforms.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+* Supplier relationship established
+* Includes performance benchmarking
+* Supporting documentation as needed
+
+#### LTE Certified
+
+LTE certification is a superset of [128T Certified Plus](#128t-certified-plus-platform). Systems that contain an **onboard** LTE adapater are required to pass certification from the carrier.
+
+LTE certified platforms have the following:
+* Has obtained PTCRB (or GCF in Europe) and mobile-carrier specific certification in the specified platform
```
Comment (ptimmons) [inline docs/about_supported_platforms.md:39]: define acronyms or link to their definitions elsewhere

File: docs/about_supported_platforms.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+:::note
+certification is tied to a specific LTE card, hw platform, and 128T sw version
+:::
+* If certification is not present, SIM card acquisition may be difficult from carrier in large quantities
+
+### 128T Certified Platform
+
+The 128T Certification program provides a designation of Certified for certain platforms. Certified platforms have the following:
```
Comment (ptimmons) [inline docs/about_supported_platforms.md:47]: ... have the following characteristics:

File: docs/about_supported_platforms.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+* Supporting documentation as needed
+
+### 128T Qualified Platform
+
+The 128T Certification program provides a designation of Qualified for certain platforms. 
+:::note
+128 Technology strongly recommends certification for any build in a production deployment. When a qualified platform is used, certification should be done by the end operator or a third party. See [compatible platform support](#support-for-128t-on-compatible-platforms). 
+:::
+
+Qualified platforms have the following:
```
Comment (ptimmons) [inline docs/about_supported_platforms.md:61]: ... have the following characteristics:

File: docs/about_supported_platforms.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+## Minimum Platform Specifications
+
+These are the minimum platform specifications for running the 128T software.
+
+### Router
+
+* 4C Intel x86 processor
+* 8GB ECC Memory
+* 64GB SSD
+* 2 DPDK enabled NIC ports (standalone)
```
Comment (ptimmons) [inline docs/about_supported_platforms.md:106]: one is enough for 'router on a stick' deployments

File: docs/about_supported_platforms.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+* 4 DPDK enabled NIC ports (HA)
+* 1 (1G) Management port
```
Comment (ptimmons) [inline docs/about_supported_platforms.md:108]: not necessary?

File: docs/about_supported_platforms.md [hunk 8]
```diff
@@ -0,0 +1,123 @@
...
+
+Inclusion in the 128T Certification Program is based on a given platforms ability to continuously meet certain test and support criteria. New platforms are added to the program at 128 Technology's discretion. For details on how to get new platforms in to the 128T certification program, or further information regarding the program, please contact 128 Technology.
+
+### 128T Certified Plus Platform
+
+The 128T Certification program provides a designation of Certified Plus for certain platforms. Certified Plus platforms have the following:
+
+* 128T software is continuously tested on the platform for performance and functionality
+* Complete build, including BIOS, DMI information, and labeling controlled by 128 Technology specification
+* May include commitment to keep up with 3rd party supply chain and engineering changes (end of life parts, BIOS changes, etc.)
```
Comment (ptimmons) [inline docs/about_supported_platforms.md:29]: third-party

Comment (ptimmons) [reply docs/about_supported_platforms.md:29]: (this 3rd party -> third-party change should be made throughout)

File: docs/about_supported_platforms.md (hunk 7 above)
Comment (MichaelBaj) [reply docs/about_supported_platforms.md:108]: Are you referring to the management port not being necessary, or listing the minimum requirements?
Regarding minimum requirements, IMO that is one of the main points of this document, is to educate everyone what is required in order to run 128T software.

Comment (ptimmons) [reply docs/about_supported_platforms.md]: The management port is not necessary. We support management over forwarding interfaces. The way this reads you'd need three ethernet interfaces to run one router. You can run it on one.

Comment (ptimmons) [reply docs/about_supported_platforms.md]: Perhaps say:
* 1 DPDK enabled NIC port for standalone systems (two recommended)
* 2 DPDK enabled NIC ports for HA systems (three recommended)
* 1 dedicated NIC port for HA synchronization
* (optional) 1 dedicated NIC port for out-of-band management

File: docs/about_supported_platforms.md [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+
+Inclusion in the 128T Certification Program is based on a given platforms ability to continuously meet certain test and support criteria. New platforms are added to the program at 128 Technology's discretion. For details on how to get new platforms in to the 128T certification program, or further information regarding the program, please [contact](https://www.128technology.com/contact) 128 Technology.
+
+### 128T Certified Plus Platform
+
+The 128T Certification program provides a designation of Certified Plus for certain platforms. Certified Plus platforms have the following characteristics:
+
+* 128T software is continuously tested on the platform for performance and functionality
+* Complete build, including BIOS, DMI information, and labeling controlled by 128 Technology specification
+* May include commitment to keep up with third-party supply chain and engineering changes (end of life parts, BIOS changes, etc.)
```
[the remark this answers is not in the document]
Comment (MichaelBaj) [reply docs/about_supported_platforms.md:30]: Are you specifically referring to "engineering changes".  to that end, this means that as the 128T software evolves, adds additional functionality, etc., we ensure that future versions of 128T software continue to run on the specified platform.

[the remark this answers is not in the document]
Comment (reidstidolph) [reply docs/about_supported_platforms.md:30]: > May include commitment to keep up with third-party supply chain and engineering changes (end of life parts, BIOS changes, etc.)

The explanation of engineering changes makes sense (although I may change to say "128T software changes"). I was actually referring more to "keeping up with third party supply chain". Is that just to say that when the platform vendor changes their platform, we are going to incorporate it?

[the remark this answers is not in the document]
Comment (ptimmons) [reply docs/about_supported_platforms.md:30]: I interpreted the "engineering changes" bit as the hardware vendor's engineering changes (ECOs), not our own software changes.

[the remark this answers is not in the document]
Comment (MichaelBaj) [reply docs/about_supported_platforms.md:30]: This language originated from Marisa. I changed "engineering" to "product" - hopefully that is less ambiguous.

**Pull request merged**